extensions/datasource/deployment/src/main/java/io/quarkus/datasource/deployment/devservices/DevServicesDatasourceProcessor.java

```diff
@@ -50,7 +51,8 @@ public class DevServicesDatasourceProcessor {
 
     // list of devservices properties we should not check for restart
     // see issue #30390
-    private static final Set<String> EXCLUDED_PROPERTIES = Set.of("quarkus.datasource.devservices.enabled");
+    private static final Set<String> EXCLUDED_PROPERTIES = Set.of("quarkus.datasource.devservices.enabled",
+            "quarkus.datasource.devservices.reuse");
```

Comment:
This looks unrelated? Please at least move to a separate commit and explain the rationale in the PR description + commit message.